Documentation/Features/PoA/PoA_voting.md

```diff
@@ -0,0 +1,41 @@
+### PoA Voting
+
+Voting is a feature that allows federation members to vote on something. At the moment of this document creation federation members can vote to add a new federation member, remove federation member, add hash to whitelisted hashes store or remove hash from there. Whitelisted hashes are being interpreted by other components. For example smart contracts feature uses whitelisted hashes as hashes of smart contracts allowed on the network. 
```

Comment:
>as hashes of smart contracts 
as hashes of smart contracts templates ...

Reply:
those are hashes of smart contracts though, not templates. we just call them templates but user is not supposed to edit them